### PR TITLE
Build arm64 image

### PR DESCRIPTION
**Pull request by nurse**
With QEMU and binfmt_misc explained as below
https://medium.com/@martin.grigorov/building-linux-packages-for-different-cpu-architectures-with-docker-and-qemu-d29e4ebc9fa5

### CODE REVIEW

**Comment by mrkn** [comment]
@nurse Do you want to build the master-debug on arm64?

File: .circleci/config.yml [hunk 1]
```diff
@@ -51,6 +50,8 @@ commands:
       - run:
           name: Build docker image
           command: |
+            docker run --rm --privileged multiarch/qemu-user-static:register --reset
+            rake docker:build ruby_version=<< parameters.ruby_version >> image_name_suffix=<< parameters.image_name_suffix >><<# parameters.nightly >> nightly=yes<</ parameters.nightly >>
```
Comment (mrkn) [inline .circleci/config.yml:54]: Why do you duplicate this line?

File: Dockerfile-arm64 [hunk 2]
```diff
@@ -0,0 +1,67 @@
+FROM multiarch/ubuntu-core:arm64-bionic
+
+ENV LANG C.UTF-8
+ENV DEBIAN_FRONTEND noninteractive
+
+ADD ruby_build_deps.txt /tmp/
+
+RUN /bin/ls
+RUN ls
```
Comment (mrkn) [inline Dockerfile-arm64:9]: Are these two `RUN`s of `ls` commands necessary?

File: Dockerfile-arm64 [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+RUN set -ex && \
+    \
+    apt-get update
```
Comment (mrkn) [inline Dockerfile-arm64:12]: Why this single `apt-get update` is necessary?

File: Dockerfile-arm64 [hunk 4]
```diff
@@ -0,0 +1,67 @@
+FROM multiarch/ubuntu-core:arm64-bionic
```
Comment (mrkn) [inline Dockerfile-arm64:1]: If the difference between `Dockerfile` and `Dockerfile-arm64` is only the base image, we can integrate them by supplying the base image name using `--build-arg`.

File: Rakefile [hunk 5]
```diff
@@ -47,4 +46,18 @@ namespace :docker do
...
+    suffix = ENV["image_name_suffix"]
+    unless File.directory?("tmp/ruby")
+      FileUtils.mkdir_p("tmp/ruby")
+      IO.write('tmp/ruby/.keep', '')
+    end
+    # amd64
+    build_image(ruby_version, suffix, nil)
+
+    # arm64
+    build_image(ruby_version, suffix, 'arm64')
```
Comment (mrkn) [inline Rakefile:61]: I guess the cause of the build timeout of `master-debug` job is building two images in a single job.
Separating jobs to build images of arm64 architecture presumably resolve the build timeout.

**Comment by nurse** [comment]
Sure, I separated the job

**Comment by mrkn** [comment]
@nurse Thank you!

**Pull request merged**